src/schema/basic_classes.yaml

```diff
@@ -25,14 +36,70 @@ classes:
       - description
       - alternative_identifiers
       - type
+
   OntologyClass:
+    aliases: ["OntologyClass"]
```

Comment:
If I ask a blunt question it doesn't mean I think your idea was crazy. Just want to succinctly start some conversations.

- Why are you asserting the class name an alias?
- why are you using [short list] notation? I don't think we have any precedent for that in nmdc-schema

Reply:
@turbomam - I toyed with the idea of changing the name of the Class ("Class" is redundant and inconsistent in the context of this schema, but after some research -- I checked into Biolink, and other LinkML schemas, and I see that its a common name for this kind of class).  I will remove the alias for now. 

short list notation is a convention I use in other models, happy to remove the list.